src/community/wps-download/src/main/java/org/geoserver/wps/gs/download/RasterDownload.java

```diff
@@ -236,6 +240,29 @@ public Resource execute(String mimeType, final ProgressListener progressListener
                 throw new WPSException("The reader did not return any data for current input "
                         + "parameters. It normally means there is nothing there, or the data got filtered out by the ROI or filter");
             }
+            
+            //
+            // STEP 0 - Check for bands, select only those specified
+            //
+            
+            if (bandIndices!=null && bandIndices.length>0){                
```

Comment:
"if you are not on Eclipse it's really just the Java official ones, with 100 chars long lines" and spaces, not tabs, used for indentation. No tabs at all. Anywhere. Please.